php/class-photonfill.php

```diff
@@ -526,6 +581,33 @@ private function get_img_src( $attachment_id, $size = null, $default = false ) {
 		}
 
 		/**
+		 * Manipulate a external url img src
+		 * @param string $img_url
+		 * @param array. This should always be an array of breakpoint width and height
+		 * @param boolean. Should this be the default srcset for the img element.
+		 */
```

Comment:
This phpdoc has incorrectly formatted @param and is missing a @return
For example: 
`@param array $size Size. This should always be an array of breakpoint width and height. Default false.`